CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing to this project
+
+Please take a moment to review this document in order to make the contribution
+process easy and effective for everyone involved.
+
+## Using the issue tracker
+
+The issue tracker is the preferred channel for [bug reports](#bug-reports),
+[features requests](#feature-requests) and [submitting pull
+requests](#pull-requests).
+
+<a name="bugs"></a>
```

Comment:
What does this do?

Reply:
Ah, should have been removed. It was there to be able to link to a section in the file, but it works without it